### DISCUSSION TITLE
Creality K1 Max on OrcaSlicer - How to connect to the klipper interface?

### DISCUSSION THREAD

**Discussion by Honeydabco** [OrcaSlicer/OrcaSlicer#1962] [General]
I just got a creality K1 max and decided to use OrcaSlicer as my slicer. It works great, my only issue is I can't seem to get the printer connected directly to the slicer. I opened the connections tab and put the IP address that I use for my web URL to access the K1 Max, however it just tells me that I need to update OctoPrint to 1.1.0 or higher. The K1 max uses the standard Klipper interface which is what I have chosen. Anyone know how to fix this so the klipper interface will show up under my Devices tab? Thanks!

**Comment by darco415** [comment]
I use Fluidd to access the webui and it works fine w/Orca...Mainsail is another option

**Comment by platonastza** [comment]
Can confirm that works just fine on my stock K1.
Enter IP , hit Ok , Ignore any error messages.

When u click device will open ther printer just fine .

**Reply by woltim99** [reply]
but problem with uploading gcode from orca slicer not solved, i'm also have this problem on my FlyingBear Ghost 5 with klipper, it can't upload files, but i can operate in device tab

**Reply by EvilGeniusJay** [reply]
Same issue here, Can connect device, but print option fails with a generic 404 error message.

**Comment by platonastza** [comment]
Well...installed fluid and as i checked...if i add the port with the ip address it works.
Just send a print from orca directly to printer.

**Reply by Pruddel** [reply]
thanks i forgot to add the port wich gave me the error.

**Reply by mathojojo** [reply]
And what is the port?

**Reply by platonastza** [reply]
for rooted printers , 4408 for Fluid or 4409 for mainsail.

**Comment by MarioLCLobato** [comment]
Same thing is happening to me, but looking at this video on YouTube you can print it, you save the g-code and then import it: https://youtu.be/uCsFvai5kMo?si=tOASPteIbzqZ14hp
